health/health_log.c

```diff
@@ -145,6 +146,10 @@ inline void health_alarm_log_save(RRDHOST *host, ALARM_ENTRY *ae) {
                             , ae->new_value
                             , ae->old_value
                             , (uint64_t)ae->last_repeat
+
+                            , (ae->class)?ae->class:"Unknown"
```

Comment:
I saw in other PR the usage of the word `Unknown`, and I am seeing it here in this PR too, considering that we are using a lot this word, I will give a suggestion for we do like we do inside [`health_config.c`](https://github.com/netdata/netdata/blob/master/health/health_config.c#L7-L29), this will help the compiler to optimize the code and I also think we will have a better organization for keywords.

Reply:
Oks! with https://github.com/netdata/netdata/pull/10850/commits/0918653d9dd04ac94ef0b8968bc10db6e7a61cce